### DISCUSSION TITLE
Lazer doesn't register input from opentabletdriver on linux

### DISCUSSION THREAD

**Discussion by archhaze24** [ppy/osu#28909] [Q&A] [answered]
I use opentabletdriver externally (not in game) because I need to use smoothing filters. After today's update, my tablet stopped working in osu while i'm using absolute mode (it worked on the previous version). 
Switching to artist mode temporarily solved the issue, but whenever I put pen away from the tablet, and then put it back, lazer stops registering inputs from tablet completely until i move my mouse. Does anybody have the same issue?
I'm on Fedora Silverblue 40, Gnome 46, wayland session.

**Comment by archhaze24** [comment, marked as answer]
solved an issue for the absolute mode by putting env var SDL_VIDEODRIVER=wayland

**Reply by salihefee** [reply]
thanks goat

**Reply by VoxArts** [reply]
thanks champ
